bin/jam/args.ts

```diff
@@ -131,6 +140,18 @@ export function parseArgs(input: string[], withRelPath: (v: string) => string):
         },
       };
     }
+    case Command.Export: {
+      const data = parseSharedOptions(args, withRelPath);
+      const outputDir = args._.shift() ?? withRelPath("export");
```

Comment:
IMHO there shouldn't be a default, we should rather bail.
If you however decide to keep the default it should be reflected in help via:
```
[output-directory]
```